### DISCUSSION TITLE
Progress sync and bookmarks sync via cloud storage

### DISCUSSION THREAD

**Discussion by qlonik** [koreader/koreader#10645] [Q&A]
I found that it is possible to synchronize reading statistics by adding a custom cloud storage (e.g. WebDav). Is it possible to synchronize reading progress using that predefined cloud storage and the folder used by reading statistics? I also found this open issue https://github.com/koreader/koreader/issues/4587 about synchronizing booksmarks. So it doesn't seem possible to do the same thing for bookmarks, isn't it? If this feature is not available, would you be open if I attempted to implement something like this and open PR?

**Comment by HammyHavoc** [comment]
Would be cool to see this, especially as my family already use a Nextcloud server, so WebDAV could be neat.
